language.md

```diff
@@ -531,6 +531,10 @@ Adds two values. If both are integers, then they are summed. If the left is a
 date and the right is an integer, a new date is produced that is later by the
 number of seconds in the integer value.
 
+If the both are lists of the same type, the union is produced. If the first is
+a list and the second is the same type as the contents of the list, a list with
+the additional item is added.
```

Comment:
Technically, I think Shesmu defines _groups_ of singletons and sets, in that addition/subtraction is defined and multiplication isn't. But I'm not sure we want to get into abstract algebra, I think talking about arithmetic operations (on singletons) and set operations (on lists) is fine.

Reply:
Calling the feature "arithmetic" is rather confusing. At first glance the term made me expect some kind of R-style vector addition/subtraction. "Arithmetic of sets" is too easily confused with "arithmetic sets" which are something different altogether. If I understand correctly, it would be clearer to call this "set algebra" or "set operations" -- where in effect, singletons are promoted to sets with a single element, and union/intersection operations work as usual.

Technically, I think Shesmu defines groups of singletons and sets, in that addition/subtraction is defined and multiplication isn't. But I'm not sure we want to get into abstract algebra, I think talking about arithmetic operations (on singletons) and set operations (on lists) is fine.

(Sorry for re-posting, deleted previous comments by accident)

Reply:
I have turned this documentation into a table. "Arithmetic" is unfortunately used like this in a lot of language grammars (to make it distinct from the logical operations). I've removed the word and renamed this all to binary operations where appropriate.

Reply:
That's better, thanks.